doc/INSTALLING.md

```diff
@@ -1,22 +1,27 @@
 ### Installing for production
+
 The project is on npm at https://www.npmjs.com/package/clappr
 
 `npm install clappr --save-dev`
 
 You should specify the base url for where the assets are located using the `baseUrl` option:
+
 ```javascript
-  var player = new Clappr.Player({
-  	source: "http://your.video/here.mp4",
-	baseUrl: "http://example.com/assets/clappr"
-  });
+var player = new Clappr.Player({
+  source: "http://your.video/here.mp4",
+  baseUrl: "http://example.com/assets/clappr"
+});
 ```
+
 In the above case clappr will expect all of the [assets (in the dist folder)](https://github.com/clappr/clappr/tree/master/dist) to be accessible at "http://example.com/assets/clappr".
 You need to arrange for the assets to be located at `baseUrl` during your build process.
 
 #### Installing for [webpack](https://webpack.github.io/)
+
 By default webpack will look at the `main` field in `package.json` and use the built version of the project. If this is all you want there is nothing else for you to do.
 
 If you would like to build the project yourself into your project during your build process then add the following to your webpack config:
+
```

Comment:
Try to avoid adding extra empty lines before or after code blocks. It only increases the file length, but with no big benefit, since the block start or end line contains only the ``` markers (and the language, optionally).